libraries/chain/util/smt_token.cpp

```diff
@@ -70,6 +70,23 @@ fc::optional< time_point_sec > last_emission_time( const database& db, const ass
    return {};
 }
 
+namespace unit_target {
+
+void verify_accounts( database& db, const flat_map< unit_target_type, uint16_t >& units )
```

Comment:
This can be placed in `smt_evaluator.cpp` directly. It is only used in the evaluator. No need to declare in a header. If is also directly tied to evaluation of the setup operation.